pallets/bitacross/src/lib.rs

```diff
@@ -70,7 +70,7 @@ pub mod pallet {
 		AdminSet { new_admin: Option<T::AccountId> },
 		RelayerAdded { who: Identity },
 		RelayerRemoved { who: Identity },
-		BtcWalletGenerated { pub_key: PubKey },
+		BtcWalletGenerated { pub_key: PubKey, account_id: T::AccountId },
```

Comment:
Some events are already present on Litmus network so I assume if anybody ever decides to resync workers this wouldn't work.

Reply:
We can make it backwards compatible if this scenario is likely to happen. We would need to keep the old functionality and based on the parentchain block number (the block on which we upgrade its runtime) we use events or extrinsics.

it can be done in a separate issue afterwards if we think we need this, we just need to know the block number

Reply:
Is event related to the block height/hash? If so it should be automatically taken care of - haven't tested it though.

Either way I think it's a small issue (for our current setup)

Reply:
I'm curious to know how to the process to resync the workers looks like. Now that I read your comment, I think I misunderstood the issue